ImputationPipeline/PCARE.wdl

```diff
@@ -111,7 +109,12 @@ task ComputeRiskValue {
 
         full_risk['combined_risk_score'] = (~{prs_beta}*full_risk.SCORE1_SUM + ~{fam_hist_beta}*full_risk.fam_hist + 
                                             ~{pc1_beta}*full_risk.PC1 + ~{pc2_beta}*full_risk.PC2)
+
+        full_risk['risk_determination'] = full_risk['combined_risk_score'].apply(
+            lambda score: 'low' if score < 19.69 else ('high' if score > 20.38 else 'average'))
```

Comment:
Can make these input parameters

Reply:
yes, let's do that

Reply:
"low", "high", and "average" are the agreed upon terms?

Reply:
Yes, that's what the VA uses and what I've documented in the mapping doc